arch/arm/soc/nordic_nrf5/include/nrf5_common.h

```diff
@@ -61,6 +61,13 @@
 #define NRF52_IRQ_I2S_IRQn                     37
 #define NRF52_IRQ_FPU_IRQn                     38
 
+/**
+ * @file Interrupt numbers specific to NRF52840 SoC
+ *
+ * The interrupt numbers below are specific to only NRF52840 SoC
+ */
+#define NRF52840_IRQ_USBD_IRQn                 39
```

Comment:
I thought this would be confusing and went with NRF52840. But what you say about future chipsets makes sense. I'll change it to just NRF52 and remove the comments too.